packages/models-library/src/models_library/api_schemas_webserver/product.py

```diff
@@ -48,10 +48,15 @@ class GetProduct(OutputSchema):
     credits_per_usd: NonNegativeDecimal | None
 
 
+class ExtraCreditsUsdRangeInt(ConstrainedInt):
+    ge = 0
+    lt = 200
```

Comment:
Q: why 200? 
was that some decision? I am a bit confused, and wondering if this should be hard-coded at all.

Reply:
yes. NK